nest-tax-backend/src/admin/admin.service.ts

```diff
@@ -201,28 +206,33 @@ export class AdminService {
       )
 
     await Promise.all(
-      norisData.map(async (elem) => {
-        birthNumbersResult.push(elem.ICO_RC)
+      norisData.map(async (norisItem) => {
+        birthNumbersResult.push(norisItem.ICO_RC)
         const taxExists = await this.prismaService.tax.findFirst({
           where: {
             year: +data.year,
             taxPayer: {
-              birthNumber: elem.ICO_RC,
+              birthNumber: norisItem.ICO_RC,
             },
           },
         })
         if (!taxExists) {
-          const { userData, dataFromNoris } =
-            await this.insertTaxPayerDataToDatabase(elem, data.year)
+          const userData = await this.insertTaxPayerDataToDatabase(
+            norisItem,
+            data.year,
+          )
           const userFromCityAccount =
             userDataFromCityAccount[userData.birthNumber] || null
           if (userFromCityAccount !== null) {
             const bloomreachTracker =
               await this.bloomreachService.trackEventTax(
                 {
-                  amount: currency(dataFromNoris.dan_spolu.replace(',', '.'))
+                  amount: currency(norisItem.dan_spolu.replace(',', '.'))
                     .intValue,
                   year: +data.year,
+                  deliveryMethod: transformDeliveryMethodToDatabaseType(
```

Comment:
```suggestion
                  delivery_method: transformDeliveryMethodToDatabaseType(
```
in other bloomreach events we are using this notation